vra7/resource_vra7_deployment.go

```diff
@@ -397,6 +399,12 @@ func resourceVra7DeploymentUpdate(d *schema.ResourceData, meta interface{}) erro
 			}
 		}
 	}
+
+	// the description and reasons cannot be updated without any valid day-2 opearation
+	if (d.HasChange("description") || d.HasChange("reasons")) && (!d.HasChange("lease_days") || !d.HasChange("resource_configuration")) {
+		return fmt.Errorf("Updating only description and/or reasons is not supported. You can update them during any supported Day-2 actions")
+	}
```

Comment:
Please check the behavior for no change to `lease_days`, but has change to `resource_configuration`.

Reply:
Yes, had to change the condition. It was invoking reconfigure and updating desc but also throwing error. 
Thanks!